### DISCUSSION TITLE
Visual Indicator that the playback is out of sync

### DISCUSSION THREAD

**Discussion by nick-potts** [f1multiviewer/issue-tracker#118] [Feature requests]
Show a visual UI element when the player is too far off the target latency.

i.e. the player is at 5s, the target is 30s. You click it, and it does the same action as pressing L - i.e. brings the player closer to 30s.

**Comment by JJWatMyself** [comment]
I like this very much. 

Sometimes you have no idea a feed is a long way from target and needs your attention. I sometimes find that the play speed isn't kicking into the slow (0.8) or fast (1.2) mode. So pressing L one, two or three times is needed to get it close and for play speed to then do the adjustments.

**Comment by f1multiviewer** [comment]
Some improvements for this have been shipped in the debug information. "Skipping to live" is still something that's quite buggy in Bitmovin, so for now, I'll have to do more research to find a good workaround for that.

**Comment by nick-potts** [comment]
Realistically, I don't think a user should have to open a hidden debug menu to find out a player is significantly out of sync.

**Reply by f1multiviewer** [reply]
I mean, I don't disagree, but "debug data" is really just what I named it initially, and that screen is vital for adjusting your sync to begin with. That being said, I'm working on a proper UI, but that's quite complex (as the screens can be quite small, and there are quite a few settings already). Improvements are coming though :)
